src/celestia/celestiacore.cpp

```diff
@@ -1880,7 +1880,7 @@ void CelestiaCore::start(double t)
         // using the KdeAlerter in runScript would create an infinite loop,
         // break it here by resetting config->initScriptFile:
         fs::path filename(config->initScriptFile);
-        config->initScriptFile = {};
+        config->initScriptFile = "";
```

Comment:
what errors do you have with `{}`?

Reply:
celestiacore.cpp(1902,35): error C2593: 'operator =' is ambiguous
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include\filesystem(737,15): message : could be 'std::filesystem::path &std::filesystem::path::operator =(std::filesystem::path::string_type &&) noexcept'
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.28.29910\include\filesystem(701,15): message : or       'std::filesystem::path &std::filesystem::path::operator =(std::filesystem::path &&) noexcept'

Reply:
could you leave a comment then so we won't change it back again? something like `don't change "" to {} or MSVC2019 throw error C2593`.
and probably better to use `fs::path()` instead of `""`.

Reply:
fixed